sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/CaseService.java

```diff
@@ -53,6 +53,30 @@ public Case buildEditGeneratedCaseForPositivePathogenTestResult() {
         .build();
   }
 
+  public Case buildGeneratedCaseForOnePerson(
```

Comment:
what's the difference between this method and existing one: buildGeneratedCase() ?